src/vs/workbench/services/search/common/textSearchManager.ts

```diff
@@ -29,60 +29,73 @@ export class TextSearchManager {
 	constructor(private query: ITextQuery, private provider: TextSearchProvider, private fileUtils: IFileUtils, private processType: ITextSearchStats['type']) { }
 
 	search(onProgress: (matches: IFileMatch[]) => void, token: CancellationToken): Promise<ISearchCompleteStats> {
-		const folderQueries = this.query.folderQueries || [];
 		const tokenSource = new CancellationTokenSource(token);
 
 		return new Promise<ISearchCompleteStats>((resolve, reject) => {
 			this.collector = new TextSearchResultsCollector(onProgress);
 
 			let isCanceled = false;
-			const onResult = (result: TextSearchResult, folderIdx: number) => {
+			const onResult = (result: TextSearchResult) => {
 				if (isCanceled) {
 					return;
 				}
 
 				if (!this.isLimitHit) {
 					const resultSize = this.resultSize(result);
-					if (extensionResultIsMatch(result) && typeof this.query.maxResults === 'number' && this.resultCount + resultSize > this.query.maxResults) {
+					if (
+						extensionResultIsMatch(result) &&
+						typeof this.query.maxResults === 'number' &&
+						this.resultCount + resultSize > this.query.maxResults
+					) {
 						this.isLimitHit = true;
 						isCanceled = true;
 						tokenSource.cancel();
 
-						result = this.trimResultToSize(result, this.query.maxResults - this.resultCount);
+						result = this.trimResultToSize(
+							result,
+							this.query.maxResults - this.resultCount,
+						);
 					}
 
 					const newResultSize = this.resultSize(result);
 					this.resultCount += newResultSize;
 					if (newResultSize > 0 || !extensionResultIsMatch(result)) {
-						this.collector!.add(result, folderIdx);
+						this.collector!.add(result, 0); // Using 0 as a default folder index
```

Comment:
```suggestion
						this.collector!.add(result, 0); // MEMBRANE: Using 0 as a default folder index
```

We're using this prefix to quickly identify our changes